packages/integration-tests/suites/replay/errors/errorsInSession/test.ts

```diff
@@ -89,7 +90,7 @@ sentryTest(
     const url = await getLocalTestPath({ testDir: __dirname });
 
     await page.goto(url);
-    await page.click('#go-background');
```

Comment:
This could probably sometimes lead to a second flush, which we didn't actually want/need. It will flush on page visit anyhow!